src/maliput_viz/plugins/maliput_mesh_converter.cc

```diff
@@ -0,0 +1,189 @@
+#include <maliput/common/maliput_abort.h>
+#include <maliput/utility/mesh.h>
+
+using namespace maliput::utility::mesh;
+
+namespace maliput {
+namespace viz {
+namespace mesh {
+namespace {
+
+// Sorts a vector of tuples composed of 3D points and an index in a polar and
+// counterclockwise way.
+//
+// Points in `unordered_vector` are supposed to be on the same plane. Based on
+// that, the mid point of all of them is computed and then used as a center of
+// coordinates. Angles are computed using that center and the compared to sort
+// the vector.
+// @returns A std::vector<std::tuple<ignition::math::Vector3d, int>> with the
+// same items of `unordered_vector` but sorted in a polar and counterclockwise
+// order.
+// @throws std::runtime_error When `unordered_vector`' size is zero.
+std::vector<std::tuple<ignition::math::Vector3d, int>> PolarSort(
+    const std::vector<std::tuple<ignition::math::Vector3d, int>>& unordered_vector) {
+  MALIPUT_DEMAND(unordered_vector.size() > 0);
+
+  std::vector<std::tuple<ignition::math::Vector3d, int>> ordered_vector = unordered_vector;
+
+  // Computes the center.
+  std::vector<ignition::math::Vector3d> points;
+  ignition::math::Vector3d center;
+  for (const std::tuple<ignition::math::Vector3d, int>& vertex_index : unordered_vector) {
+    center += std::get<0>(vertex_index);
+  }
+  center /= static_cast<double>(points.size());
+
+  // TODO(agalbachicar)    This polar sort should be done with respect to the
+  //                       plane of all the points in the face instead of the
+  //                       plane z=0. We need the normal of the plane to do
+  //                       so.
+  std::sort(
+      ordered_vector.begin(), ordered_vector.end(),
+      [center](const std::tuple<ignition::math::Vector3d, int>& a, const std::tuple<ignition::math::Vector3d, int>& b) {
+        double a_y = std::atan2(std::get<0>(a).Y() - center.Y(), std::get<0>(a).X() - center.X());
+        double b_y = std::atan2(std::get<0>(b).Y() - center.Y(), std::get<0>(b).X() - center.X());
+        if (a_y < 0) a_y += 2 * M_PI;
+        if (b_y < 0) b_y += 2 * M_PI;
+        return a_y < b_y;
+      });
+  return ordered_vector;
+}
+
+// \brief Generates a unique name for a mesh.
+// \param baseName The base name. It will be appended with "_ID".
+// \return A string formatted like: @p baseName + "_" + ID , where ID is a
+// increasing unsigned integer.
+const std::string GenerateUniqueMeshName(const std::string& baseName) {
+  static uint counter = 0;
+  return baseName + "_" + std::to_string(counter++);
+}
+
+}  // namespace
+
+std::unique_ptr<ignition::common::Mesh> Convert(const std::string& name, const GeoMesh& geo_mesh) {
+  // Checks before actually creating the mesh.
+  if (geo_mesh.num_vertices() < 3 && geo_mesh.faces().size() < 1) {
+    return nullptr;
+  }
+
+  auto mesh = std::make_unique<ignition::common::Mesh>();
+  // TODO(agalbachicar):    ignition::rendering and ignition::common do not
```

Comment:
Shall we create / migrate a ticket for this TODO? Is it still relevant?

Reply:
I left it as documentation. And updated the link.
Done.